packages/@guardian/source-foundations/src/colour/palette.tsx

```diff
@@ -90,6 +92,11 @@ const colors = {
 		'#ABC2C9', //specialReport-500
 		'#E4E5E8', //specialReport-700
 		'#EFF1F2', //specialReport-800
+		'#2B2B2A', //specialReportAlt-100
```

Comment:
from our conversation in chat I thought we were putting these in the `reds` array - or I miss a final decision on leaving them in here?

Reply:
Yeah you missed that bit

Reply:
sounds good 👍 - no strong opinion about where they live